velox/exec/MergeJoin.cpp

```diff
@@ -564,6 +578,31 @@ RowVectorPtr MergeJoin::getOutput() {
 
         // No rows survived the filter. Get more rows.
         continue;
+      } else if (isAntiJoin(joinType_)) {
```

Comment:
could we move this logic to a helper function, and comment what you are doing? This function is already pretty long and hard to follow.

Reply:
@pedroerp Yes. Added filterOutputForAntiJoin method.